docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -12,9 +12,18 @@ Returns search results for an <<eql,Event Query Language (EQL)>> query.
 EQL assumes each document in a data stream or index corresponds to an
 event.
 
+////
 [source,console]
 ----
-GET /my-index-000001/_eql/search
+DELETE /_data_stream/*
+DELETE /_index_template/*
+----
+// TEARDOWN
+////
```

Comment:
I'll replace this with a reusable teardown once https://github.com/elastic/elasticsearch/pull/70831 is merged.